### PR TITLE
Expanded audio widget

### PR DESCRIPTION
**Pull request by emord**
This adds the expanded audio widget

buddy @proteusvacuum

### CODE REVIEW

File: src/javaRosa/plugin.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ define([
         audio: 'fa fa-volume-up',
         video: 'fa fa-video-camera',
         'video-inline': 'fa fa-play',
+        'expanded-audio': 'fa fa-volume-up',
```
Comment (emord) [inline src/javaRosa/plugin.js]: Didn't find a great icon for this. open to ideas though

**Comment by millerdev** [comment]
Test failures look real
```
  2 failing
  1) The javaRosa plugin with multiple languages should generate bulk multi-line translation with user-friendly newlines:
      expected 'label\tdefault_en\tdefault_hin\timage_en\timage_hin\taudio_en\taudio_hin\tvideo_en\tvideo_hin\tvideo-inline_en\tvideo-inline_hin\texpanded-audio_en\texpanded-audio_hin\nquestion1-label\t"First ""line\nSecond"" line\nThird line"\tHindu trans\t\t\t\t\t\t\t\t\t\t' to equal 'label\tdefault_en\tdefault_hin\taudio_en\taudio_hin\timage_en\timage_hin\tvideo_en\tvideo_hin\tvideo-inline_en\tvideo-inline_hin\texpanded-audio_en\texpanded-audio_hin\nquestion1-label\t"First ""line\nSecond"" line\nThird line"\tHindu trans\t\t\t\t\t\t\t\t\t\t'
      + expected - actual
      -label	default_en	default_hin	image_en	image_hin	audio_en	audio_hin	video_en	video_hin	video-inline_en	video-inline_hin	expanded-audio_en	expanded-audio_hin
      +label	default_en	default_hin	audio_en	audio_hin	image_en	image_hin	video_en	video_hin	video-inline_en	video-inline_hin	expanded-audio_en	expanded-audio_hin
       question1-label	"First ""line
       Second"" line
       Third line"	Hindu trans										
      
  
  2) The javaRosa plugin with multiple languages should escape all languages when generating bulk translations:
      expected 'label\tdefault_en\tdefault_hin\timage_en\timage_hin\taudio_en\taudio_hin\tvideo_en\tvideo_hin\tvideo-inline_en\tvideo-inline_hin\texpanded-audio_en\texpanded-audio_hin\ntext-label\t"""Text"\t"""Text"\t\t\t\t\t\t\t\t\t\t' to equal 'label\tdefault_en\tdefault_hin\taudio_en\taudio_hin\timage_en\timage_hin\tvideo_en\tvideo_hin\tvideo-inline_en\tvideo-inline_hin\texpanded-audio_en\texpanded-audio_hin\ntext-label\t"""Text"\t"""Text"\t\t\t\t\t\t\t\t\t\t'
      + expected - actual
      -label	default_en	default_hin	image_en	image_hin	audio_en	audio_hin	video_en	video_hin	video-inline_en	video-inline_hin	expanded-audio_en	expanded-audio_hin
      +label	default_en	default_hin	audio_en	audio_hin	image_en	image_hin	video_en	video_hin	video-inline_en	video-inline_hin	expanded-audio_en	expanded-audio_hin
       text-label	"""Text"	"""Text"
```

**Comment by emord** [comment]
:white_check_mark:

File: src/javaRosa/util.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ define([
     util
 ) {
     var SUPPORTED_MEDIA_TYPES = ['image', 'audio', 'video', 'video-inline', 'expanded-audio'],
-        ITEXT_TYPES = _.union(['default'], SUPPORTED_MEDIA_TYPES),
+        ITEXT_TYPES = ['default', 'audio', 'image', 'video', 'video-inline', 'expanded-audio'],
```
Comment (millerdev) [inline src/javaRosa/util.js]: Curious if you know why this broke the tests? Would `['default'].concat(SUPPORTED_MEDIA_TYPES)` have worked instead?

Comment (emord) [inline src/javaRosa/util.js]: The ordering mattered for some tests. I think I could have just changed what the export expected, but didn't think too hard about it. At least it's one less hard coded array

**Pull request merged**